run_test.py

```
@@ -1,5 +1,8 @@
 import os
-os.environ['SECRET_KEY'] = 'dummy_key'
-import pytest
+
+os.environ["SECRET_KEY"] = "dummy_key"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)